pkg/controller/certificate-shim/gateways/controller_test.go

```diff
@@ -63,7 +63,7 @@ func Test_controller_Register(t *testing.T) {
 			name: "gateway is re-queued when an 'Updated' event is received for this gateway",
 			givenCall: func(t *testing.T, _ cmclient.Interface, c gwclient.Interface) {
 				// We can't use the gateway-api fake.NewSimpleClientset due to
-				// Gateway being pluralized as "gatewaies" instead of
+				// Gateway being pluralized as "gateway" -"y" +"ies" instead of
```

Comment:
I think this does a better job of explaining what's going wrong than spelling out the non word directly. But as w/ all of my changes, I'm happy to drop things...